engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/KafkaAvroBaseTransformer.scala

```diff
@@ -25,14 +25,16 @@ trait KafkaAvroBaseTransformer[T] extends SingleInputGenericNodeTransformation[T
 
   def schemaRegistryProvider: SchemaRegistryProvider
 
+  def topicSelectionStrategy(schemaRegistryClient: SchemaRegistryClient): TopicSelectionStrategy = new AllTopicsSelectionStrategy(schemaRegistryClient)
```

Comment:
If we make it def topicSelectionStrategy: TopicSelectionStrategy,
and pass schemaRegistryClient as parameter to strategy method, we'll enable having strategy as field in subclasses

Reply:
woops good point :+1: , I forgotten that this `getTopicParam` is invoken quite often.

Reply:
moved as param